apps/planning-suite-kit/contracts/PlanningKit.sol

```diff
@@ -247,7 +247,7 @@ contract PlanningKit is KitBase {
         address root = msg.sender;
         addressBook.initialize();
         projects.initialize(registry, vault, token);
-        dotVoting.initialize(addressBook, token, 50 * PCT256, 0, 1 minutes);
+        dotVoting.initialize(token, 50 * PCT256, 0, 1 minutes);
```

Comment:
This seems a really nice move!, but I think it is a good idea to also perform the same change to our production template at `kits/planning-suite/contracts/`

Reply:
@Quazia you want to handle that?

Reply:
It was an easy change to make; never mind. See most recent commit; I changed it.